src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -67,11 +67,15 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
         """
         self._future = future
         self._streams = streams
-
+        self._stage = stage
+        self._starting_pos = {'x': area[0], 'y': area[1]}  # left, top
```

Comment:
Two problems with this:
* You should use `normalized_area` if you want to be sure it's the left,top
* It's not needed to start at the very corner, but only close enough to make sure that the top left of the image will contain this point. I think this is a more general limitation of the current procedure though. To rephrase it, `area` should indicate the area imaged. As the stream has a FoV, the movement of the stage can be smaller. You can think of it as a border around of the area. So the movement area should be "area - (smallest_FoV/2 - overlap_distance)". The overlap distance is removed to represents, pessimistically, the precision of the stage. For small FoVs (eg FM), that is not going to have much effect, but for large FoV (eg EM at low mag), it may end-up going for example from 4x4 -> 3x3 tiles, which could be quite a save in time.

No need to address this point now. Let's do this in a separate task.